database/schema/12-emoney_inflation.sql

```diff
@@ -2,11 +2,10 @@
 
 CREATE TABLE emoney_inflation
```

Comment:
Once thing I just realized: since we're adding a new module's support, would you mind also including the new Hasura metadata inside the repo? It would be nice to track it as well.

Reply:
thanks! fixed and pushed. btw I changed the file name **emoney_inflation.sql** to **emoney.sql** as there are more customized modules coming, I will group the SQL CMDs in this file.

Reply:
hi @RiccardoM , I added the [yaml file](https://github.com/forbole/bdjuno/pull/204/files#diff-11eedd60179e520e66a421e69f9b4b5289f4f285a489a66cd35de72cdc914e81) , could you please check if I've done it right ? thanks!